crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -2727,6 +2727,66 @@ pub unsafe fn vzip2q_f64(a: float64x2_t, b: float64x2_t) -> float64x2_t {
     simd_shuffle2(a, b, [1, 3])
 }
 
+/// Unsigned Absolute difference and Accumulate Long
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(uabd))]
+pub unsafe fn vabal_high_u8(a: uint16x8_t, b: uint8x16_t, c: uint8x16_t) -> uint16x8_t {
+    let d: uint8x16_t = vabdq_u8(b, c);
+    let e: uint8x8_t = simd_shuffle8(d, d, [8, 9, 10, 11, 12, 13, 14, 15]);
+    simd_add(a, simd_cast(e))
```

Comment:
This implementation is incorrect. You need to extract the high part of each input and then call `vabal_u8`. See the clang implementation in `arm_neon.h`.

Reply:
Actually I think the issue here is the same as the other one. The `simd_cast` should be changed to do zero-extension instead of sign-extension.

Reply:
> Actually I think the issue here is the same as the other one. The `simd_cast` should be changed to do zero-extension instead of sign-extension.

Done. Thanks